app/Console/Commands/UpdateGivenStreams.php

```diff
@@ -6,23 +6,34 @@
 use App\Models\Stream;
 use App\Services\Youtube\StreamData;
 use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Builder;
 
 class UpdateGivenStreams extends Command
 {
-    protected $signature = 'larastreamers:update-streams';
+    protected $signature = 'larastreamers:update-streams {--soon-live-only}';
 
     protected $description = 'Update all today/upcoming streams';
 
     public function handle(): int
     {
-        $streams = Stream::query()->upcoming()->get()->keyBy('youtube_id');
+        $streams = Stream::query()
```

Comment:
Currently, we also update "old" streams. Do we need that?

Reply:
Not sure if I get you right.
i don't see a need to update streams that aren't live or upcoming

the condition is needed to ensure we only hit the API as early as possible